docs/user/dql/functions.rst

```diff
@@ -1557,6 +1594,40 @@ Example::
     +------------------------------+
 
 
+UNIX_TIMESTAMP
+--------------
+
+Description
+>>>>>>>>>>>
+
+Usage: Converts given argument to Unix time (seconds since January 1st, 1970 at 00:00:00 UTC). If no argument given, it returns current Unix time.
+The date argument may be a DATE, DATETIME, or TIMESTAMP string, or a number in YYMMDD, YYMMDDhhmmss, YYYYMMDD, or YYYYMMDDhhmmss format. If the argument includes a time part, it may optionally include a fractional seconds part.
+If argument is in invalid format or outside of range 1970-01-01 00:00:00 - 3001-01-18 23:59:59.999999 (0 to 32536771199.999999 epoch time), function returns NULL.
+You can use `FROM_UNIXTIME`_ to do reverse conversion.
+
+Argument type: <NONE>/DOUBLE/DATE/DATETIME/TIMESTAMP
+
+Return type: DOUBLE
+
+Examples::
+
+    os> select UNIX_TIMESTAMP(20771122143845)
+    fetched rows / total rows = 1/1
+    +----------------------------------+
+    | UNIX_TIMESTAMP(20771122143845)   |
+    |----------------------------------|
+    | 3404817525.0                     |
+    +----------------------------------+
+
+    os> select UNIX_TIMESTAMP(TIMESTAMP('1996-11-15 17:05:42'))
```

Comment:
MySQL shows this as returning without a floating point.
```
mysql> select UNIX_TIMESTAMP(TIMESTAMP('1996-11-15 17:05:42'));
+--------------------------------------------------+
| UNIX_TIMESTAMP(TIMESTAMP('1996-11-15 17:05:42')) |
+--------------------------------------------------+
|                                        848077542 |
+--------------------------------------------------+
1 row in set (0.00 sec)
```

Reply:
It is added by the `double` serializer, it is out of my control. We can discuss to fix this globally in scope of another fix.